packages/internal-test-helpers/lib/test-cases/abstract.js

```diff
@@ -88,11 +98,17 @@ export default class AbstractTestCase {
   }
 
   click(selector) {
-    return this.$(selector).click();
+    let element;
+    if (typeof selector === 'string') {
+      element = this.element.querySelector(selector);
+    } else {
+      element = selector;
+    }
+    return element.click();
   }
 
   textValue() {
-    return this.$().text();
+    return this.element ? this.element.textContent : '';
```

Comment:
does anything break without the guard?

Reply:
Yes, some of the `visit` tests with `shouldRender: false` throw without the guard. Specifically they are trying to confirm that nothing has rendered. 

I could remove the guard, and update those handful of tests to simply use `assert.equal(document.getElementById('qunit-fixture').children.length, 0)`.

Reply:
I see.. maybe put that in an `assertNothingRendered` method or something. Perhaps we can just assert `this.element = null`.. but I am not sure if that semantically works for all the tests.

Reply:
It was only one test after the other visit test refactors that I did so I just put an inline assertion there...